### PR TITLE
➕Update nexmo, and more deps to latest versions

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -2,7 +2,7 @@
 max-line-length=120
 filename=*.py,*.py.dev
 exclude=./env/*,./config/*,./fab*,.git/*,static/bower/*,sitestatic/bower/,*/gen/*
-ignore=E501,F405,T003,E203,W503
+ignore=E501,F405,T003,E203,W503,T001
```
[the remark this answers is not in the document]
Comment (norkans7) [reply setup.cfg]: It is "found print"

[the remark this answers is not in the document]
Comment (nicpottier) [reply setup.cfg]: What was newly triggering that?

[the remark this answers is not in the document]
Comment (norkans7) [reply setup.cfg]: new flake8

**Comment by norkans7** [comment]
The /n in tests is to adapt to new nexmo library that check for a file if the private key is a string without it. The reason for all this is that cryptography<2 was not being installed on PY3.7 when @rowanseymour tried that so decided to update nexmo to the latest, and more deps as possible

**Comment by norkans7** [comment]
Also I merge https://github.com/nyaruka/rapidpro/pull/2058 in this to avoid any possible conflicts

**Comment by norkans7** [comment]
Closing this to make smaller progressive PRs like #2062 for each package

**Pull request closed without merging**